CDP4WebServices.API/CDP4WebServices.API.csproj

```diff
@@ -27,6 +27,7 @@
   
   <ItemGroup>
     <ProjectReference Include="..\CDP4Authentication\CDP4Authentication.csproj" />
+    <ProjectReference Include="..\CDP4DatabaseAuthentication\CDP4DatabaseAuthentication.csproj" />
```

Comment:
i doubt this needs to be here. I think we load plugins at runtime, so we do not need to reference the projects